### PR TITLE
create tests/conftest.py. Browsers: Chrome, Firefox, MsEdge.

### PR DESCRIPTION
**Pull request by kit-91-daniil**

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from msedge.selenium_tools import Edge
+from selene.support.shared import browser as selene_browser
+from selenium.webdriver.chrome.options import Options
+from webdriver_manager.chrome import ChromeDriverManager
+from webdriver_manager.firefox import GeckoDriverManager
+from webdriver_manager.microsoft import EdgeChromiumDriverManager
+
+
+def pytest_addoption(parser):
+    parser.addoption("--browser_name", action="store", default="chrome", help="Choose browser: chrome or firefox")
```
Comment (relaxing-observer) [inline tests/conftest.py]: "Chrome or firefox or edge" in help maybe?

File: tests/conftest.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+
+@pytest.fixture(scope='function', autouse=True)
```
Comment (relaxing-observer) [inline tests/conftest.py]: Consider changing scope to 'module' or 'package'

**Review by relaxing-observer — CHANGES_REQUESTED** [review]

**Review by relaxing-observer — APPROVED** [review]

**Review by yefim-mironyuk — APPROVED** [review]
Ochen' kruto. Molodec

**Review by TA-user — APPROVED** [review]
looks good

**Pull request merged**